tutorials/convert_lit_models.md

```diff
@@ -1,23 +1,118 @@
-## Converting Lit-GPT weights to HuggingFace Transformers
+## Converting Lit-GPT weights to Hugging Face Transformers
 
-Lit-GPT weights need to be converted to a format that HuggingFace understands with a [conversion script](../scripts/convert_lit_checkpoint.py) before our scripts can run.
+Lit-GPT weights need to be converted to a format that Hugging Face understands with a [conversion script](../scripts/convert_lit_checkpoint.py) before our scripts can run.
 
-We provide a helpful script to convert models Lit-GPT models back to their equivalent HuggingFace Transformers format:
+We provide a helpful script to convert models Lit-GPT models back to their equivalent Hugging Face Transformers format:
 
 ```sh
 python scripts/convert_lit_checkpoint.py \
     --checkpoint_path path/to/litgpt/model.pth \
-    --output_path where/to/save/the/converted.ckpt \
+    --output_path out/converted.ckpt \
     --config_path path/to/litgpt/config.json
 ```
 
 These paths are just placeholders, you will need to customize them based on which finetuning or pretraining script you ran and it's configuration.
 
+### Loading converted Lit-GPT checkpoints into transformers
+
+If you want to load the converted checkpoints into a `transformers` model, please make sure you copied the original `config.json` file into the folder that contains the `converted.ckpt` file saved via `--output_path` above.
```

Comment:
In my opinion, when we download a hf checkpoint, we should include all files in the folder including config.json, and carry it around so that later on, you can convert back without hunting down these missing file (including tokenizer config etc.): https://github.com/Lightning-AI/lit-gpt/issues/923

I have struggled with this too.


But anyway, it's great to describe this now, even if it's cumbersome.

Reply:
I feel the same way. The config.json is extremely small so why not. Let me add a PR to address this.

Reply:
I would merge this PR before trying to address that